Microsoft Academic.js

```diff
@@ -399,21 +384,27 @@ var testCases = [
 						"tag": "Mathematics"
 					},
 					{
-						"tag": "Mean Curvature"
+						"tag": "Mean curvature"
+					},
+					{
+						"tag": "Mean curvature flow"
+					},
+					{
+						"tag": "Principal curvature"
 					},
 					{
-						"tag": "Mean Curvature Flow"
+						"tag": "Riemann curvature tensor"
 					},
 					{
-						"tag": "Scalar Curvature"
+						"tag": "Scalar curvature"
 					},
 					{
-						"tag": "Topology"
+						"tag": "Sectional curvature"
 					}
 				],
 				"notes": [
 					{
-						"note": "<p>Data sources found by Microsoft Academic search engine:</p><a href=\"https://graphics.stanford.edu/courses/cs468-03-fall/Papers/cohen_normalcycle.pdf\">https://graphics.stanford.edu/courses/cs468-03-fall/Papers/cohen_normalcycle.pdf</a><br/><a href=\"https://dpt-info.u-strasbg.fr/~sauvage/Recherche/GT_geom_diff/CM03.pdf\">https://dpt-info.u-strasbg.fr/~sauvage/Recherche/GT_geom_diff/CM03.pdf</a><br/><a href=\"http://dblp.uni-trier.de/db/conf/compgeom/compgeom2003.html#Cohen-SteinerM03\">http://dblp.uni-trier.de/db/conf/compgeom/compgeom2003.html#Cohen-SteinerM03</a><br/><a href=\"http://dl.acm.org/citation.cfm?doid=777792.777839\">http://dl.acm.org/citation.cfm?doid=777792.777839</a><br/><a href=\"http://doi.acm.org/10.1145/777792.777839\">http://doi.acm.org/10.1145/777792.777839</a><br/><a href=\"http://portal.acm.org/citation.cfm?doid=777792.777839\">http://portal.acm.org/citation.cfm?doid=777792.777839</a><br/>"
+						"note": "<p>Data sources found by Microsoft Academic search engine:</p><a href=\"http://math.univ-lyon1.fr/~morvan/SCANPUBLICATIONS/cohensteinermorvansocg2003.pdf\">http://math.univ-lyon1.fr/~morvan/SCANPUBLICATIONS/cohensteinermorvansocg2003.pdf</a><br/><a href=\"http://www.cs.jhu.edu/~misha/Fall09/Steiner03.pdf\">http://www.cs.jhu.edu/~misha/Fall09/Steiner03.pdf</a><br/><a href=\"https://graphics.stanford.edu/courses/cs468-03-fall/Papers/cohen_normalcycle.pdf\">https://graphics.stanford.edu/courses/cs468-03-fall/Papers/cohen_normalcycle.pdf</a><br/><a href=\"http://portal.acm.org/citation.cfm?doid=777792.777839\">http://portal.acm.org/citation.cfm?doid=777792.777839</a><br/><a href=\"https://core.ac.uk/display/22880751\">https://core.ac.uk/display/22880751</a><br/><a href=\"https://dl.acm.org/citation.cfm?doid=777792.777839\">https://dl.acm.org/citation.cfm?doid=777792.777839</a><br/><a href=\"https://doi.acm.org/10.1145/777792.777839\">https://doi.acm.org/10.1145/777792.777839</a><br/>"
```

Comment:
Are we sure we want to keep the notes? I could see us trying to grab a PDF attachment from this information, but as a note added to every item this doesn't seem great.

Reply:
But which URL you would try for grabbing the PDF from? We certainly don't want to add a PDF from all the URLs. The one with the DOI looks best, but maybe the user does not have access to the pay-walled paper. Anyway, there is the unpaywall integration in Zotero which is doing that job nowadays for publications with a DOI.

I would be okay to drop this note therefore. Also I think that one of the main advantages of Microsoft Academic (or Google Scholar) is, that it tries to find a freely available version of the paper.